upload/swift.go

```diff
@@ -0,0 +1,85 @@
+package upload
+
+import (
+	"io/ioutil"
+	"log"
+	"path"
+	"path/filepath"
+
+	"github.com/ncw/swift"
+
+	"data-processing-spark-submit/utils"
+)
+
+type (
+	Swift struct {
+		StorageI
+		c *swift.Connection
+	}
+
+	SwiftConf struct {
+		UserName string `ini:"user_name"`
+		Password string `ini:"password"`
+		AuthURL  string `ini:"auth_url"`
+		Domain   string `ini:"domain"`
+		Region   string `ini:"region"`
+	}
+)
+
+//init storage
+func NewSwift(conf *SwiftConf) (*Swift, error) {
```

Comment:
rename this file (and the objects within from swift to ovhcloud_object_storage?

Reply:
no it's  protocol here

Reply:
Understood.